docs/modules/ROOT/pages/tutorials/getting-started/java-web.adoc

```diff
@@ -0,0 +1,53 @@
+= Getting Started on {product} with the OpenShift Web Console using Java
+
+// THIS FILE IS AUTOGENERATED
+// DO NOT EDIT MANUALLY
+
+image:logos/java.svg[role="related thumb right",alt="Programming language logo",width=120,height=120] This tutorial explains how to run applications written in the Java programming language on {product} using the OpenShift web console.
+
+If you would rather use the command line, you might want to try the xref:tutorials/getting-started/java-terminal.adoc[Getting Started with Java on the Terminal] guide instead.
+
+== Requirements
+
+This tutorial doesn't require anything else than an {product} account and a modern web browser with JavaScript enabled.
+
+== About the Application
+
+To demo how to run applications written in the https://www.java.com/en/[Java] programming language on {product}, we will use a small demo application using the https://quarkus.io/[Quarkus Framework], bundled as a container thanks to its corresponding `Dockerfile`, and ready to be used on {product}.
+
+The "Fortune in Java" application does one simple thing: when invoked on the browser, it returns a random number and a funny quote. That's all!
+
+You can browse the full source code of this application on GitLab:
+
+* https://gitlab.com/vshn/applications/fortune-java
+
+== Step 1: Deploy a Container
+
+Follow these steps to create an OpenShift project and deploy the application:
+
+. Login to {product} and open the "Developer" perspective.
+. Create a new project called "fortune-java."
+. Switch to the "Developer" perspective.
```

Comment:
Do I really have to switch to the "Developer" perspective a second time?

Reply:
Solved